src/event.rs

```diff
@@ -20,6 +20,7 @@ use term::{Term, SizeInfo, TermMode};
 use util::limit;
 use util::fmt::Red;
 use window::Window;
+use time::{self, Timespec};
```

Comment:
I think `std::time::{Instant, Duration}` should be sufficient for the usage here, and we don't need a third-party dependency. Recording click time would be

```rust
Instant::now()
```

Comparing click times would be

```rust
let now = Instant::now();
let delta /* Duration */ = now.duration_since(last_click_time);
if delta < click_threshold {
    // ..
}
```